### PR TITLE
Adding option to add retentionInDays

### PR DESCRIPTION
**Pull request by raphaelquati**
#31

### CODE REVIEW

File: src/Serilog.Sinks.AwsCloudWatch/CloudWatchSinkOptions.cs [hunk 1]
```diff
@@ -35,5 +35,10 @@ public class CloudWatchSinkOptions
         /// It's recommended to implement a custom formatter like a simple JSON formatter with various parameters included.
         /// </summary>
         public ILogEventRenderer LogEventRenderer { get; set; }
+
+        /// <summary>
+        /// Specify the RetentionInDays in RetentionPolicy used in AWS CloudWatch.
```
Comment (thoean) [inline src/Serilog.Sinks.AwsCloudWatch/CloudWatchSinkOptions.cs]: The [put retention policy](http://docs.aws.amazon.com/AmazonCloudWatchLogs/latest/APIReference/API_PutRetentionPolicy.html) is fairly restricted regarding valid values. I'd prefer to have these restrictions highlighted in the comment here.

File: src/Serilog.Sinks.AwsCloudWatch/CloudWatchSinkOptions.cs [hunk 2]
```diff
@@ -35,5 +35,10 @@ public class CloudWatchSinkOptions
...
+        /// </summary>
+        public int? RetentionInDays { get; set; }
```
Comment (thoean) [inline src/Serilog.Sinks.AwsCloudWatch/CloudWatchSinkOptions.cs]: Should this be a `TimeSpan`, which is more flexible to future options? It'll also allow to make this variable exclude the word `Days`. In the actual code, you could then use `TimeSpan.TotalDays`.

**Comment by thoean** [comment]
@raphaelquati - do you intend to respond to the review, and potentially fine tune your pull request, or should we close this without merging?

**Comment by thoean** [comment]
Closing due to inactivity. We'd appreciate if you'd re-open it again as you see fit.

**Comment by justinhelgerson** [comment]
@thoean I'll pick this up as I'd like the ability to automate the retention instead of having to tweak it in the AWS Console.  I'll include your feedback in my pull.

**Pull request closed without merging**